src/actions/public/estimateGas.ts

```diff
@@ -235,21 +277,24 @@ export async function estimateGas<
       return estimate
     }
 
-    const { verificationGasLimit, callGasLimit } = (await client.request({
-      method: 'eth_estimateGas',
-      params: rpcStateOverride
-        ? [request, block ?? 'latest', rpcStateOverride]
-        : block
-          ? [request, block]
-          : [request],
-    })) as {
-      verificationGasLimit: Quantity
-      callGasLimit: Quantity
-    }
+    // For RIP-7560 transactions
```

Comment:
I think it's a case that works exclusively with the `if (!request.sender)` above. What if it shows explicitly exclusive behavior like else?